alerting/notifier/channels/kafka.go

```diff
@@ -73,6 +91,16 @@ func buildKafkaSettings(fc FactoryConfig) (*kafkaSettings, error) {
 	if settings.Details == "" {
 		settings.Details = DefaultMessageEmbed
 	}
+
```

Comment:
Password is supposed to be a secret. So please add 
```suggestion
	settings.Password = fc.DecryptFunc(context.Background(), fc.Config.SecureSettings, "password", settings.Password)
```

Reply:
Added!